packages/core/upload/tests/content-api/graphql-upload-automatic-folder.test.e2e.js

```diff
@@ -166,11 +166,11 @@ describe('Uploads folder (GraphQL)', () => {
       expect(file).toMatchObject({
         folder: {
           name: 'API Uploads (1)',
-          uid: expect.anything(),
+          pathId: expect.anything(),
```

Comment:
```suggestion
          pathId: expect.any(Number),
```

Same for a couple of instances below.